pkg/render/dex.go

```diff
@@ -43,8 +46,11 @@ const (
 	DexPort          = 5556
 	DexTLSSecretName = "tigera-dex-tls"
 	DexClientId      = "tigera-manager"
+	DexPolicyName    = networkpolicy.TigeraComponentPolicyPrefix + "allow-tigera-dex"
 )
 
+var DexEntityRule = networkpolicy.CreateEntityRule(DexNamespace, DexObjectName, DexPort)
```

Comment:
Do we need to declare this globally? Seems like its only used in one place?

Reply:
It's referenced by multiple render components - should be referenced by the tiers component in the first commit (and by more components in subsequent commits)